File: python/8-kyu/total-amount-of-points/total-amount-of-points.py
def points(games):
    total = 0
    listed_scores = [[int(x) for x in s.split(':')] for s in games]
    for score in listed_scores:
        if score[0] > score[1]:
            total += 3
        elif score[0] == score[1]:
            total += 1
        else:
            total += 0
    return total

File: javascript/8-kyu/total-amount-of-points/total-amount-of-points.js
function points(games) {
  const groupedPairs = games.map(str => str.split(':').map(Number));
  let total = 0;
  for (const [x, y] of groupedPairs) {
    if (x > y) {
      total += 3;
    } else if ( x === y) {
      total += 1;
    } else {
      total += 0;
    }
  }
  return total;
}